3.0-RELEASE-NOTES.md

```diff
@@ -62,4 +62,28 @@ methods: {
 }
 ```
 
-Please see [related code change](https://github.com/strongloop/loopback/pull/2174) here.
\ No newline at end of file
+Please see [related code change](https://github.com/strongloop/loopback/pull/2174) here.
+
+## remove unused user properties
+The following properties are removed from the common User model in 3.0:
+- credentials
+- challenges
+- status
+- created
+- lastUpdated
+
+Developers that are relying on these properties, can redefine them in `user.json` or equivalent model.json as follow: 
+```js
+//user.json
+"credentials": {
```

Comment:
Please start at the top level of model JSON file to make it more clear where to add this snippet. Also use `json` for highlighting (you can check the source of my comment to see how to do it in markdown):

```json
{
  "name": "MyUser",
  "base": "User",
  "properties": {
    "credentials": { "type": "object" },
    "challenges": { "type": "object" },
    "status": "string",
    "created": "date",
    "lastUpdated": "date"
  }
} 
```